packages/fxa-admin-server/src/scripts/audit-tokens.ts

```diff
@@ -495,6 +510,11 @@ if (require.main === module) {
     })
     .catch((error) => {
       log.error('audit-tokens', { error });
+      Sentry.captureCheckIn({
```

Comment:
We could have sentry `captureException` here if we weren't reporting these before already.